gov2/secretsmanager/common/create.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package common
```

Comment:
Maybe move this to below //snippet-sourceauthor:[gangwere] (so it would be below the new 'Purpose' section)